build/webpack.styles.js

```diff
@@ -1,27 +1,39 @@
 import ExtractTextPlugin from 'extract-text-webpack-plugin'
+// import postcssPlugins from './webpack.postcss'
 
-export default {
-  dev: {
+const styleLoaders = {
+  development: {
     test: /\.css$/,
     loaders: [
       'style-loader',
-      { loader: 'css-loader', query: { modules: true } },
-      { loader: 'postcss-loader' }
+      { loader: 'css-loader', query: { modules: true, importLoaders: 1 } },
+      { loader: 'postcss-loader'
```

Comment:
yep. Should drop this week sometime - https://github.com/postcss/postcss-loader/issues/110